ci-operator/config/openshift/assisted-image-service/openshift-assisted-image-service-release-ocm-2.5.yaml

```diff
@@ -96,6 +120,14 @@ tests:
   secret:
     mount_path: /tmp/secret
     name: assisted-image-service-codecov-token
+- as: e2e-metal-assisted
+  skip_if_only_changed: ^docs/|\.md$|^(?:.*/)?(?:\.gitignore|OWNERS|OWNERS_ALIASES|PROJECT|LICENSE)$
```

Comment:
I'd make it totally optional, or just match ``main`` branch behavior (with the additional deploy-nodes job)